sdk/Cargo.toml

```diff
@@ -160,9 +156,7 @@ web-sys = { version = "0.3.58", features = [
 
 [dev-dependencies]
 anyhow = "1.0.40"
-c2pa = { path = ".", default-features = false, features = [
-    "unstable_api",
-] } # allow integration tests to use the new API
+c2pa = { path = ".", default-features = false }
```

Comment:
Does this even need to be here at all any more?

Reply:
good call, it isn't. I removed it.